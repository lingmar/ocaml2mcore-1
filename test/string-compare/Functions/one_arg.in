let foo x = x in foo
