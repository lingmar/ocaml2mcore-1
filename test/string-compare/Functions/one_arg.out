mexpr
let foo = lam x. x in foo
